flytekit/clis/sdk_in_container/run.py

```diff
@@ -721,6 +722,17 @@ class WorkflowCommand(click.RichGroup):
 
     def __init__(self, filename: str, *args, **kwargs):
         super().__init__(*args, **kwargs)
+
+        if FileAccessProvider.is_remote(filename):
```

Comment:
Thanks! could you use file_access in flytekit context because flytekit has data persistence plugin and it can download file from different sources (s3, http, gcs, etc). 
```python
ctx = context_manager.FlyteContextManager.current_context()
if ctx.file_access.is_remote(filename):
    local_path = "."
    ctx.file_access.get(filename, local_path)
```

Reply:
And I think we should remove the file after we run it.

Reply:
btw, could you add a test [here](https://github.com/flyteorg/flytekit/blob/master/tests/flytekit/unit/cli/pyflyte/test_run.py).